controllers/app.go

```diff
@@ -3,41 +3,32 @@ package main
 import (
 	"log"
 	"todo/database"
+	"todo/database/cache"
 	"todo/models"
 
 	"github.com/gofiber/fiber/v2"
-	"github.com/gofiber/fiber/v2/middleware/basicauth"
 	"github.com/jinzhu/copier"
 )
 
 func main() {
 	app := fiber.New()
+	cache.CacheSetUp()
 	database.TaskSetUp()
 	database.UserSetUp()
 
-	app.Use("/task", basicauth.New(basicauth.Config{
```

Comment:
For the following changes of basicauth I simply removed the functionality here, and put this into a middleware file, found in Todo/middleware/middleware.go.